plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -69,6 +69,10 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_RENAME_TABLE:
             case SUPPORTS_COMMENT_ON_TABLE:
             case SUPPORTS_COMMENT_ON_COLUMN:
+                return false;
```

Comment:
The below conditions look redundant:
* SUPPORTS_CREATE_VIEW
* SUPPORTS_TRUNCATE
* SUPPORTS_DELETE
* SUPPORTS_ADD_COLUMN_WITH_COMMENT

Also, could you reorder other conditions based on `TestingConnectorBehavior`? For instance, `SUPPORTS_SET_COLUMN_TYPE ` should be closed to `SUPPORTS_ADD_COLUMN `.

Reply:
Remove redundant branches:

- SUPPORTS_CREATE_VIEW
- SUPPORTS_LIMIT_PUSHDOWN
- SUPPORTS_TOPN_PUSHDOWN

Rearrange the left, please suggest